assets/in

```diff
@@ -72,7 +73,15 @@ for branch in $fetch; do
   git branch $branch FETCH_HEAD
 done
 
+return_ref=""
+if [ -n "$tag_filter" ]; then
```

Comment:
Just as a side note: The globing works and it is covered by the tests. But if the command is tried manually, be sure to quote the wildcards to avoid the shell expand or change the wildcard expression.